shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/widget/injection/StudyPlanComponentImpl.kt

```diff
@@ -0,0 +1,22 @@
+package org.hyperskill.app.study_plan.widget.injection
+
+import org.hyperskill.app.core.injection.AppGraph
+import org.hyperskill.app.core.presentation.ActionDispatcherOptions
+import org.hyperskill.app.study_plan.widget.presentation.StudyPlanWidgetActionDispatcher
+import org.hyperskill.app.study_plan.widget.presentation.StudyPlanWidgetReducer
+import org.hyperskill.app.study_plan.widget.view.StudyPlanWidgetViewStateMapper
+
+class StudyPlanComponentImpl(private val appGraph: AppGraph) : StudyPlanWidgetComponent {
```

Comment:
I think we should rename `StudyPlanComponentImpl -> StudyPlanWidgetComponentImpl`